src/workerd/api/tsconfig.json

```diff
@@ -11,7 +11,8 @@
     },
     "composite": true,
     "declaration": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "allowJs": true
```

Comment:
allowJs added to match ts_project in wd_test – this is not needed now but we might have wd_tests using both JS + TS in the future.